.github/workflows/create-cache.yml

```diff
@@ -6,25 +6,31 @@ on:
       - main
 
 jobs:
```

Comment:
I've now update this section to be identical to the one in `ci.yml` to be as consistent as possible